src/components/UserPage.js

```diff
@@ -23,14 +26,30 @@ const UserPage = () => {
       })
       .catch(err => {
         console.log(err);
-      });
+      });*/
   }, [username]);
 
+  let greeting;
+
+  if (localStorage.getItem("username") === username) {
+    greeting = `Welcome ${username}`;
+    console.log("Welcome");
+  } else {
+    greeting = `${username} Videos`;
+    console.log(username);
+  }
+
   return (
     <>
       <Container>
         <div>
-          {videos.map(video => {
+          {/* Need to display user's username in Welcome*/}
+          <h1>{greeting}</h1>
+          <p>
+            Now that you're logged in, lets{" "}
```

Comment:
We'll want to make sure to address this one as well

Reply:
completed